pkgs/by-name/pi/pinentry-kwallet/package.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, kwalletcli }:
+stdenv.mkDerivation {
+  pname = "pinentry-kwallet";
+  version = kwalletcli.version;
+
+  phases = [ "buildPhase" ];
```

Comment:
```suggestion
```
See #28910. You can add `dontBuild = true` etc.

Reply:
Ah I didn't know about that, thanks! I now disabled all phases with `dont*`.